src/Arcus.Messaging.Abstractions.ServiceBus/MessageHandling/AzureServiceBusMessageRouter.cs

```diff
@@ -272,40 +266,14 @@ private async Task<MessageProcessingResult> TryRouteMessageWithPotentialFallback
                     }
```

Comment:
Just wondering; where do we call 'Complete' if the message was succesfully processed ?
In this method, I see calls to Abandon or to Deadletter the message, depending on the case, but I don't see a call to Complete the message, when the message was processed succesfully.
Probably done somewhere else, but maybe it makes the code more readable / understandable if Complete is called here as well ?